src/app/public/testing/phonefield/phone-field-fixture.spec.ts

```diff
@@ -70,6 +67,7 @@ const VALID_US_NUMBER = '8675555309';
       <input
         formControlName="phoneControl"
         skyPhoneFieldInput
+        [disabled]="disabled"
```

Comment:
```suggestion
        [attr.disabled]="disabled"
```

To avoid the warning `It looks like you're using the disabled attribute with a reactive form directive...`